core/server/worker/src/test/java/alluxio/worker/block/BlockStoreMetaTest.java

```diff
@@ -82,7 +82,7 @@ public void getBlockList() {
   }
 
   /**
-   * Tests the {@link BlockStoreMeta#getCapacityBytes()} method.
+   * Tests the {@link DefaultBlockStoreMeta#getCapacityBytes()} method.
```

Comment:
@dayhost fix javadoc ? (reverting `DefaultBlockStoreMeta#...` to `BlockStoreMeta#` in javadoc of this file?